packages/components/organisms/f-checkout/stories/Checkout.stories.js

```diff
@@ -45,7 +45,7 @@ export const CheckoutComponent = () => ({
             default: text('Create Guest Url', createGuestUrl)
         },
         authToken: {
-            default: text('Auth token', '')
+            default: select('Auth token', ['authToken', null], 'authToken')
```

Comment:
Any reason for this change? You won't be able to use a token because the dropdown will be empty, I believe.

Reply:
So it was the best way I could think of to be able to show guest in storybook without adding a new prop just for storybook.
If the authToken is `null` then checkout becomes guest.
This is how it comes out 
![image](https://user-images.githubusercontent.com/24740015/104320769-a8692980-54da-11eb-8e49-1980e460cee4.png)
Happy to change it if there's another way I've missed

Reply:
Discussed with Sacha, keeping it like this for now